src/features/list-view/ListView.js

```diff
@@ -56,7 +71,13 @@ class ListView extends React.PureComponent<Props> {
         const cellData = getGridCell({ columnIndex, rowIndex: rowIndex - 1 });
```

Comment:
I'm realizing what confuses me about "rowIndex: rowIndex - 1". It should be "cellIndex: rowIndex - 1". This naming is consistent with the header versus cell convention. Thoughts?

Reply:
This makes sense. It's not immediately obvious to a developer why the rowIndex - 1 exists but calling it a cell helps give more clarity.